Unix/configure

```diff
@@ -2025,10 +2073,16 @@ echo "created $fn"
 ##==============================================================================
 
 if [ "$dev" = "1" ]; then
+  mkdir -p $certsdir
+  echo "built certs dir " $certsdir 
   openssl req -x509 -sha1 -newkey rsa:2048 -days 3650 -nodes -config $fn -keyout $certsdir/omikey.pem -out $certsdir/omi.pem
   openssl x509 -outform der -in $certsdir/omi.pem -out $certsdir/omi.der
   chmod 600 $certsdir/omikey.pem
   chmod 644 $certsdir/omi.pem
+  echo "built certs in " $certsdir 
```

Comment:
Is this a duplicate of line 2077? Or should 2077 say "Building certs in dir ..." ?